src/components/Login.js

```diff
@@ -36,6 +36,10 @@ class Login extends React.Component {
   componentWillUnmount() {
     this.props.onUnload();
   }
+  
+  componentDidMount() {
+    console.log('Console from Login while componentDidMount');
```

Comment:
Review comments form the base repository for the code changes.

Reply:
Reply comments to the Owner after the fix the review comments

Reply:
Resolved